rke/k8s_rke_system_images.go

```diff
@@ -8506,6 +8550,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		// Enabled out of band post v2.6.11
+		"v1.24.10-rancher1-1": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.5.4",
+			Kubernetes:                "rancher/hyperkube:v1.24.10-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.88",
+			NginxProxy:                "rancher/rke-tools:v0.1.88",
+			CertDownloader:            "rancher/rke-tools:v0.1.88",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.88",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.21.1",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.21.1",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.21.1",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.5",
+			Flannel:                   "rancher/mirrored-coreos-flannel:v0.15.1",
+			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher6",
+			CalicoNode:                "rancher/mirrored-calico-node:v3.22.5",
+			CalicoCNI:                 "rancher/calico-cni:v3.22.5-rancher1",
+			CalicoControllers:         "rancher/mirrored-calico-kube-controllers:v3.22.5",
+			CalicoCtl:                 "rancher/mirrored-calico-ctl:v3.22.5",
+			CalicoFlexVol:             "rancher/mirrored-calico-pod2daemon-flexvol:v3.22.5",
+			CanalNode:                 "rancher/mirrored-calico-node:v3.22.5",
+			CanalCNI:                  "rancher/calico-cni:v3.22.5-rancher1",
+			CanalControllers:          "rancher/mirrored-calico-kube-controllers:v3.22.5",
+			CanalFlannel:              "rancher/mirrored-flannelcni-flannel:v0.17.0",
+			CanalFlexVol:              "rancher/mirrored-calico-pod2daemon-flexvol:v3.22.5",
+			WeaveNode:                 "weaveworks/weave-kube:2.8.1",
+			WeaveCNI:                  "weaveworks/weave-npc:2.8.1",
+			AciCniDeployContainer:     "noiro/cnideploy:5.2.3.5.1d150da",
+			AciHostContainer:          "noiro/aci-containers-host:5.2.3.5.1d150da",
+			AciOpflexContainer:        "noiro/opflex:5.2.3.5.1d150da",
+			AciMcastContainer:         "noiro/opflex:5.2.3.5.1d150da",
+			AciOpenvSwitchContainer:   "noiro/openvswitch:5.2.3.5.1d150da",
+			AciControllerContainer:    "noiro/aci-containers-controller:5.2.3.5.1d150da",
+			AciGbpServerContainer:     "noiro/gbp-server:5.2.3.5.1d150da",
+			AciOpflexServerContainer:  "noiro/opflex-server:5.2.3.5.1d150da",
+			PodInfraContainer:         "rancher/mirrored-pause:3.6",
+			Ingress:                   "rancher/nginx-ingress-controller:nginx-1.5.1-rancher1",
```

Comment:
The actual published tag is [nginx-1.5.1-rancher2](https://hub.docker.com/layers/rancher/nginx-ingress-controller/nginx-1.5.1-rancher2/images/sha256-035e995c229de4f9b0571736c6937d8a0fbea27fa63cdf13baaad633cff5f92a?context=explore) so this needs to be updated here.